PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -56,11 +66,39 @@ class PsfPowershellScriptRunner
 	//RunStartingScript should return an error only if stopOnScriptError is true
 	void RunStartingScript()
 	{
+		LogString("StartingScript commandString", this->startingScriptInformation.commandString.c_str());
+		LogString("StartingScript currentDirectory", this->startingScriptInformation.currentDirectory.c_str());
+		if (this->startingScriptInformation.runInVirtualEnvironment)

```

Comment:
runInVirtualEnvironment is taken away.  The script will always run in the virtual environment.  The logging and logic for runInVirtualEnvironment can be removed.

Reply:
OK. But leaving the logging in; just like other log entries in the method, they only appear if using the debug build and are helpful if troubleshooting scripts.

Reply:
By the way I am assuming you meant running in the VE is fixed.  The json still provides a selectable option on this, right?  Sometimes we NEED the script to run external; most times internal.

Reply:
Hey Tim.

Leaving the debugger code in is fine.

I'm sorry but it was a decision made higher up.  Scripts are not allowed ever to run outside of the environment.  Letting scripts run outside of the environment breaks the MSIX promise that everything is contained.  There is no option anymore to have scripts run outside the VE.  Sorry about that.

Reply:
OK. Doesn't accomplish what they think it does and will make more resistance to IT Pros converting to MSIX.  But, that's not my problem either. Needs to be communicated out, though.